lib/reverse_sentence.rb

```diff
@@ -1,4 +1,60 @@
 # A method to reverse the words in a sentence, in place.
+
+# Share and explain the time and space complexities for your solution.
+# Time complexity:
+# O(n) + O(n**2) + O(n)
+# Space complexity:
+# O(1)
+
+def string_reverse(my_string)
+  return my_string if my_string.nil? || my_string == ""
+  i = 0
+  j = my_string.length - 1
+
+  while i < j
+    placeholder = my_string[i]
+    my_string[i] = my_string[j]
+    my_string[j] = placeholder
+    i += 1
+    j -= 1
+  end
+  return my_string
+end
+
+
 def reverse_sentence(my_sentence)
-  raise NotImplementedError
+  return my_sentence if my_sentence.nil? || my_sentence == ""
```

Comment:
Same comments on this method as I had on reverse_words: https://github.com/Ada-C10/reverse_words/pull/19#issuecomment-439174853